src/test/ui/derives/issue-91550.stderr

```diff
@@ -8,7 +8,7 @@ LL | struct Value(u32);
    | doesn't satisfy `Value: Hash`
 ...
 LL |     hs.insert(Value(0));
-   |        ^^^^^^ method cannot be called on `HashSet<Value>` due to unsatisfied trait bounds
```

Comment:
I'm not sure that this is necessarily an improvement, it doesn't hurt the quality of the error dramatically or anything like that but it's probably a little worse-off.

Reply:
Yeah, I think this is one of the examples where the quality is slightly worse, one the challenges with this has been balancing the consequences of the change. Do you think this is a blocker or do the benefits of the change outweigh this?

Reply:
I don't think this is a blocker, in each of the tests that have changed like this, all of the information that was lost is available elsewhere in the diagnostic.